Issue 29097: adjusted ids in entities and entitylist to be PascalCase

MrMothman opened the pull request:
<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- The text between the arrows are comments - they will not be visible on your PR. -->

## About the PR
<!-- What did you change in this PR? -->
I adjusted files inside of entities and entitylist so that the ids: feild is all in pascal case: the entities list is below
./Markers/clientsideclone.yml:  id: clientsideclone
./Markers/construction_ghost.yml:  id: constructionghost
./Markers/drag_shadow.yml:  id: dragshadow
./Markers/hover_entity.yml:  id: hoverentity
./Objects/Fun/dice.yml:  id: d20Dice
./Objects/Fun/dice.yml:  id: d12Dice
./Objects/Fun/dice.yml:  id: d10Dice
./Objects/Fun/dice.yml:  id: d8Dice
./Objects/Fun/dice.yml:  id: d6Dice
./Objects/Fun/dice.yml:  id: d4Dice
./Structures/hydro_tray.yml:  id: hydroponicsTray
./Structures/Machines/Computers/computers.yml:  id: computerBodyScanner
./Structures/soil.yml:  id: hydroponicsSoil
./Structures/Storage/filing_cabinets.yml:  id: filingCabinet
./Structures/Storage/filing_cabinets.yml:  id: filingCabinetTall
./Structures/Storage/filing_cabinets.yml:  id: filingCabinetDrawer
./Structures/Storage/filing_cabinets.yml:  id: filingCabinetRandom
./Structures/Storage/filing_cabinets.yml:  id: filingCabinetTallRandom
./Structures/Storage/filing_cabinets.yml:  id: filingCabinetDrawerRandom
./Objects/Tools/t-ray.yml:  id: trayScanner
## Why / Balance
<!-- Why was it changed? Link any discussions or issues here. Please discuss how this would affect game balance. -->
https://github.com/space-wizards/space-station-14/pull/28693
Also, I found this to be an easier task and want to get use to working within this project before I take on bigger projects.
Also, Cases matter to keep the same:
NOTE: I didn't changed dice_bag.yml or any behavior ids:, I'm not sure if we want to change those but if we do It is an easy fix.
## Technical details
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
## Media
<!-- 
PRs which make ingame changes (adding clothing, items, new features, etc) are required to have media attached that showcase the changes.
Small fixes/refactors are exempt.
Any media may be used in SS14 progress reports, with clear credit given.

If you're unsure whether your PR will require media, ask a maintainer.

Check the box below to confirm that you have in fact seen this (put an X in the brackets, like [X]):
-->

- [X ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
<!--
List any breaking changes, including namespace, public class/method/field changes, prototype renames; and provide instructions for fixing them. This will be pasted in #codebase-changes.
-->

**Changelog**
<!--
Make players aware of new features and changes that could affect how they play the game by adding a Changelog entry. Please read the Changelog guidelines located at: https://docs.spacestation14.io/en/getting-started/pr-guideline#changelog
-->
- tweak: ids: fields to match PascalCase usage else where.
<!--
Make sure to take this Changelog template out of the comment block in order for it to show up.
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->

TheShuEd reviewed (changes requested):
you also need to change the names of all the relationships to the changed entities.

to give you an example:
https://github.com/search?q=repo%3Aspace-wizards%2Fspace-station-14%20clientsideclone&type=code

lzk228 reviewed (changes requested):
also you need to put all changed entities' ids in /Resources/migration.yml

deltanedas commented:
this also doesnt account for the few ids with _ in them

MrMothman commented:
Thank you I assumed it was somewhere else as well @lzk228  @TheShuEd : ) @deltanedas  all the ones with _ in the name are behavior Ids: and i didn't know if we also wanted those changed. I didn't do behavior ids: or the dice_bag.yml because those were pretty weird and i didn't know if their was a reason.

lzk228 commented:
ok leave it as it is for now, just do what i and Ed said

MrMothman commented:
i ran find . -type f -exec grep -H -f p.txt {} + > FilesToAdjust.txt
at root with FilesToAdjust.txt containing all the names of ids: I've changed and had many occurrences of those so I changed all of those and the mirgration.yml actually didn't have any occurrences in it: )

lzk228 commented:
you dont need to change maps if you write all changes in migration.yml

MrMothman commented:
ahhh the more you learn then. I am sorry about that
today I learned

TheShuEd commented:
you still need to add changes to the migration because the forks have their maps still using those items.  Forcing ALL projects to rewrite these changes manually is a very bad idea.

MrMothman commented:
I got that sorry I also had to figure out the error that was caused when I changed the entityList .ymls

MrMothman commented:
One day latter and I got all the checks: )

lzk228 commented:
Ed tfym approved
@MrMothman you should revert all map changes

lzk228 commented on Resources/migration.yml:
```diff
@@ -348,3 +348,28 @@ DoorRemoteFirefight: null
+filingCabinetDrawerRandom: FilingCabinetDrawerRandom
+computerBodyScanner: ComputerBodyScanner
+
+
+
+
+
+
+
```
remove those extra lines

SlamBamActionman commented:
Hello! Thank you for your contribution.

This was discussed internally and due to concerns about downstream forks, merge conflicts and migration issues we're not interested in large-scale entity ID refactors at the moment, so I will be closing this PR for the time being.
